### PR TITLE
Move ballot example into src/app/example-contracts

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by chriseth** [comment]
Restarting test, not sure if this is an issue with infrastructure or with how the example file is loaded.

**Comment by axic** [comment]
@chriseth fixed!

File: src/app/example-contracts.js [hunk 1]
```diff
@@ -1,30 +1,4 @@
...
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-var multi = function (func) { return func.toString().match(/[^]*\/\*([^]*)\*\/\}$/)[1]; };
-
-var BALLOT_EXAMPLE = multi(function () { /*pragma solidity ^0.4.0;
+var ballot = `pragma solidity ^0.4.0;
```
Comment (chriseth) [inline src/app/example-contracts.js]: This will not work on non-es6 browsers. Please see the debugger PR.

Comment (axic) [inline src/app/example-contracts.js]: The babel processing in #273 will sort this too.

**Comment by axic** [comment]
@yann300 do you have an idea why the test fails?

**Pull request merged**